cider/tests/nextgen/compiler/NextgenCompilerTest.cpp

```diff
@@ -131,7 +129,7 @@ class CiderNextgenCompilerTestBase : public CiderTestBase {
 };
 
 TEST_F(CiderNextgenCompilerTestBase, integerFilterTest) {
-  assertQueryArrow("SELECT col_1 + col_2 FROM test WHERE col_1 < col_2");
+  assertQueryArrow("SELECT col_1 + col_2 FROM test WHERE col_1 <= col_2");
```

Comment:
Why need this change?

Reply:
due to data pattern, just to make sure the query has result